efficientdet/gradient_checkpointing/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2018 OpenAI (http://openai.com)
```

Comment:
Hmm, we are not able to import code with such copyright.

Reply:
@mingxingtan memory_saving_gradients is open source codebase (openai) ,cant we keep the copyright?

Reply:
I see. That's too bad. Is there any workaround? In the meantime I may try to ask the authors if there's a way to remove the copyright keeping mentioning their names. How can l I explain why it has to be removed?

Reply:
@mingxingtan
What if keep all the libraries as a separate repository? Then you don't need to copy them here, you would be able to use them with "pip install" and import. Is it a valid workaround?

Reply:
Great, thanks. Waiting for your resolution then